test/src/subclient/index.js

```diff
@@ -23,11 +23,6 @@ class SubClient {
             "Incentivized": null
           }
         },
-        "MessageNonce": "u64",
```

Comment:
Looks like you need to pull in from `main` :)

Reply:
Ah this must have gotten cut out during the merge. Added it back in https://github.com/Snowfork/polkadot-ethereum/pull/266/commits/58c393183e6cbe7fc00e483d4bd5a007e27598b0